script/test/cmd/cmd_test.go

```diff
@@ -0,0 +1,248 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/fsouza/go-dockerclient/external/github.com/Sirupsen/logrus"
+	"github.com/joho/godotenv"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"reflect"
+	"strings"
+	"testing"
+)
+
+func fileNotExist(path string) bool {
+	return findExecutable(path) != nil
+}
+func findExecutable(file string) error {
+	d, err := os.Stat(file)
+	if err != nil {
+		return err
+	}
+	if m := d.Mode(); !m.IsDir() {
+		return nil
+	}
+	return os.ErrPermission
+}
+
+func binary(cmd string) string {
+	path, err := exec.LookPath(cmd)
+	if err != nil || fileNotExist(cmd) {
+		home := os.Getenv("HOME")
+		if home == "" {
+			fmt.Println("No $HOME environment variable found")
+		}
+	}
+	return path
+}
+
+// Kompose command
+func Kompose(command, provider, dockerComposeFile, result string) (cmd, output string) {
+	b, _ := ioutil.ReadFile(result)
+	file := strings.Split(dockerComposeFile[9:], "/")
+	output = string(b)
+	cmdName := "kompose"
+	cmdArgs := []string{command, "--provider", provider, "-f", dockerComposeFile, "--stdout", "-j"}
+	out, err := exec.Command(cmdName, cmdArgs...).CombinedOutput()
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Error Cmd", err, file[0])
+		os.Exit(1)
+	}
+	cmd = string(out)
+	return cmd, output
+}
+
+// Compare compares two json files
+func Compare(output, cmd, dockerComposeFile, provider string) error {
+	file := strings.Split(dockerComposeFile[9:], "/")
```

Comment:
@containscafeine I didn't find any function in the `filepath` package which will do the something similar to this, although I have modified the `strings.Split(dockerComposeFile[9:], "/")` to `strings.Split(dockerComposeFile, "/")` and its working as it should.